emojiHeatmapper/script.js

```diff
@@ -121,6 +128,25 @@ loklakFetcher.getTweets(query, function(tweets) {
                 geometry: point
             }));
         }
+        emotions_array.sort();
+        emotion_array = jQuery.unique( emotions_array );
+
+        //Loading the sentiment
+        $(document).ready(function() {
+            var listItems= "";
+            listItems += "<h3> Sentiment(s) of " + query + " is/are ";
+            for (var i = 0; i < emotion_array.length - 1; i++) {
+                listItems += emotion_array[i] + ", ";
+            }
+            if (i == emotion_array.length - 1) {
+                listItems += emotion_array[i] + ".";
+            }
```

Comment:
Done @djmgit . And `var emotion_data = emotion_array.join(", ") + "."` this part may end with `,.` , i think so.